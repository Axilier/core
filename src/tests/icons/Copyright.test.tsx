import React from 'react';
import { shallow } from 'enzyme';
import { Copyright } from '../../index';

const copyright = shallow(<Copyright />);

it('should render correctly', () => {
    expect(copyright).toMatchSnapshot();
});
