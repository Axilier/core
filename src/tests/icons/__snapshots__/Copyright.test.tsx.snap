// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<svg
  className="undefined icon"
  clipRule="evenodd"
  fillRule="evenodd"
  strokeLinejoin="round"
  strokeMiterlimit={2}
  viewBox="0 0 16 16"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M8 0c4.415 0 8 3.585 8 8s-3.585 8-8 8-8-3.585-8-8 3.585-8 8-8zm0 1.52A6.484 6.484 0 0114.48 8 6.484 6.484 0 018 14.48 6.484 6.484 0 011.52 8 6.484 6.484 0 018 1.52z"
    fill="#fff"
  />
  <text
    fill="#fff"
    fontFamily="'ArialMT','Arial',sans-serif"
    fontSize={12}
    transform="translate(-4.671 4.061)"
    x={8.277}
    y={8.234}
  >
    C
  </text>
</svg>
`;
